src/components/materialization/SourceCapture/SourceCaptureChip.tsx

```diff
@@ -16,32 +17,26 @@ function SourceCaptureChip() {
 
     const saving = useSourceCaptureStore((state) => state.saving);
 
-    const disabled = saving || formActive;
-
-    if (!sourceCapture) {
-        return (
-            <Chip
-                color="info"
-                disabled={disabled}
-                label={
-                    <FormattedMessage id="workflows.sourceCapture.selected.none" />
-                }
-            />
-        );
-    }
-
     return (
         <Chip
-            color="success"
-            disabled={disabled}
-            label={sourceCapture}
-            sx={{
-                maxWidth: '50%',
-            }}
-            onDelete={async () => {
-                setSourceCapture(null);
-                await updateDraft(null);
-            }}
+            color={sourceCapture ? 'success' : 'info'}
+            disabled={saving || formActive}
+            label={
+                sourceCapture ??
+                intl.formatMessage({
+                    id: 'workflows.sourceCapture.selected.none',
+                })
+            }
+            variant="outlined"
+            style={{ maxWidth: '50%' }}
+            onDelete={
+                sourceCapture
+                    ? async () => {
+                          setSourceCapture(null);
+                          await updateDraft(null);
+                      }
+                    : undefined
+            }
```

Comment:
Updated to `outlined` to get better styling
Also, there was no reason to have two different returns... not sure why that was done.